SU2_CFD/src/driver_direct_multizone.cpp

```diff
@@ -207,6 +207,15 @@ void CMultizoneDriver::Preprocess(unsigned long TimeIter) {
     /*--- TODO: This should be generalised for an homogeneous criteria throughout the code. --------*/
     config_container[iZone]->SetExtIter(TimeIter);
 
+    /*--- Store the current physical time in the config container, as
+     this can be used for verification / MMS. This should also be more
+     general once the drivers are more stable. ---*/
+    
+    if (unsteady)
+      config_container[iZone]->SetPhysicalTime(static_cast<su2double>(TimeIter)*config_container[iZone]->GetDelta_UnstTimeND());
```

Comment:
Why static_cast?

Reply:
I guess this is to avoid a compiler warning, but @economon is better suited to comment on this.

Reply:
I only ask because most other casts in the code are C style

Reply:
No particular reason from what I recall. We can go with C style for consistency

Reply:
Actually that seems to be the safe way to do it (https://stackoverflow.com/questions/332030/when-should-static-cast-dynamic-cast-const-cast-and-reinterpret-cast-be-used) keep it please.